src/core/meshcop/commissioner.cpp

```diff
@@ -735,7 +744,7 @@ void Commissioner::HandleLeaderPetitionResponse(Coap::Message *         aMessage
     {
         if (mTransmitAttempts >= kPetitionRetryCount)
         {
-            SetState(OT_COMMISSIONER_STATE_DISABLED);
+            Stop(true);
```

Comment:
We are not in active state, so I think the commissioner should not send the keep alive message (whether to keep it self alive or stop itself gracefully).

Reply:
nit: add comment to the boolean argument of `Stop`? Like this `Stop(/* aSendKeepAlive */ false)`.

Reply:
> nit: add comment to the boolean argument of `Stop`? Like this `Stop(/* aSendKeepAlive */ false)`.

IDEs like `Clion` can actually do the job for you by showing you the argument names. 
I think the comment is helpful, but I haven't seen it been used in `openthread` code.

Reply:
Search `(/*` to find example code. The comment is only useful for arguments that can't easily tell its meaning ( such as a boolean argument).